proxy/tests/transparency.rs

```diff
@@ -3,8 +3,6 @@ use self::support::*;
 
 #[test]
 fn outbound_http1() {
-    let _ = env_logger::init();
```

Comment:
It's very useful to be able to turn on logs if debugging a failing test. Can these just become `let _ = env_logger::try_init();`?

Reply:
I updated the tests to use `try_init()` instead as you suggested.